py/janitor/site/simple.py

```diff
@@ -705,12 +705,18 @@ async def main_async(argv=None):
     public_runner = web.AppRunner(public_app)
     await private_runner.setup()
     await public_runner.setup()
+
     site = web.TCPSite(private_runner, args.host, port=args.port)
     await site.start()
-    logging.info("Listening on %s:%s", args.host, args.port)
+    logging.info("Private runner listening on %s:%s", args.host, args.port)
+
     site = web.TCPSite(public_runner, args.host, port=args.public_port)
     await site.start()
-    logging.info("Listening on %s:%s", args.host, args.public_port)
+    logging.info("Public runner listening on %s:%s", args.host, args.public_port)
+
+    if args.debugtoolbar:
```

Comment:
Why not print this on line 613?

Reply:
My thinking was to keep it with the other`logging.info`.
Move it to where its imported?

Reply:
Yeah, I think that makes most sense.

Reply:
I can, but in another comment, your saying not to have it - so Im now a little confused.
Move it or drop it?

REF: https://github.com/jelmer/janitor/pull/972#discussion_r1979659240

Reply:
Gotcha - moved.